contents/docs/tutorials/integrate-with-metabase.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Integrate PostHog with Metabase
+sidebar: Docs
+showTitle: true
+---
+
+## Objective
+
+Integrating PostHog with your Metabase instance.
+
+[Metabase](https://www.metabase.com/) is an open-source data explorer and Business Intelligence tool. 
+
+## Pre-Requisites
+
+To follow this tutorial along, you should:
+
+1. Have [deployed PostHog](/docs/deployment). 
+
+**Important Note** 
+
+Integrating with Metabase currently works only for self-hosted PostHog instances. Additionally, since Metabase does not support ClickHouse, you are only able to integrate it with your PostHog database if your PostHog instance uses PostgreSQL for event data (i.e. you're not using our Enterprise Edition). 
```

Comment:
This isn't _impossible_ as this states but it's complex. Ie you could do:

PostHog -> Some sort of datalake that Metabase supports -> Metabase